app/controllers/api/coins_controller.rb

```diff
@@ -23,9 +23,17 @@ def search_by_params
     if params[:coinSlugs].present? 
       coins = Coin.where(slug: params[:coinSlugs])
     elsif params[:name].present?
-      coins = Coin.where('upper(concat(symbol,\' \', name)) like upper(?)', "%#{params[:name]}%")
-                  .order('length(name) asc')
-                  .limit(10)
+      coins = Coin.find_by_sql("
```

Comment:
@anthotsang Just curious about how the query performance on this?

Reply:
Seems to be similar to the previous query. The EXPLAIN() for both has the same plan/costs.